client/src/main/java/com/netflix/conductor/client/http/WorkflowClient.java

```diff
@@ -357,14 +358,15 @@ public void resumeWorkflow(String workflowId) {
      * @param workflowId the id of the workflow instance
      * @param taskReferenceName the reference name of the task to be skipped
      */
-    public void skipTaskFromWorkflow(String workflowId, String taskReferenceName) {
```

Comment:
can you create an overloaded method and deprecate the existing method so we do not break backward compatibility?

Reply:
yes, sure